webrtc.html

```diff
@@ -509,33 +509,42 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>Append an <code>operation</code> to
+            the <var>operations</var> queue, where
+            an <code>operation</code> represents the asynchronous
+            execution of the called function that will fulfill the
+            promise returned by the function.</p>.
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>If there is exactly one <code>operation</code> in the
+            queue, start the operation.</p>
```

Comment:
To be clear: your definition of *operation* as "the asynchronous execution of the called function that fulfills the returned promise", clashes with the fact that all the functions having both synchronous and asynchronous parts in their [processing steps](http://w3c.github.io/webrtc-pc/#dom-peerconnection-setlocaldescription) that need to execute, and your model is not executing the synchronous parts. If you're trying to define *operation* to mean *only* the asynchronous parts of said processing models, then please don't.